telnet.c

```diff
@@ -599,8 +599,6 @@ struct ios_ops *telnet_init(char *hostport)
 		/* send intent to do and accept COM_PORT stuff */
```

Comment:
`s/do and//` in above comment?

Reply:
Good catch, that comment needs adaption. However sending a "DO" means "Dear remote side, please talk COM_PORT with me" and "WILL" means "I will talk COM_PORT". So now that "DO" gets dropped, `s/and accept//` is the thing to do.